README.md

```diff
@@ -1,189 +1,157 @@
+
+```shell
+c2patool image.jpg
+```
+
+### Detailed manifest report
+
+The `-d` option will output a detailed JSON report of the internal C2PA structure.
+
+```shell
+c2patool image.jpg  -d
+```
+
+### Previewing a manifest
+
+If a path to a JSON config file is given, the tool will generate a new manifest using the values given in the definition. By default, this will print the results to stdout. You can save the results to a file by using the `-o` flag, followed by the path of the destination file.
 
-If a path to a json config file is given,
-the tool will generate a new manifest using the values given in definition
-this will display the results but not save anything unless an output (-o) is specified
+```shell
+# output to screen
+c2patool sample/config.json
 
-```c2patool sample/config.json```
+# save to file
+c2patool sample/config.json -o results.txt
+```
 
-The config json can also be passed on the command line as string using the -c --config option
+The [manifest definition](#manifest-definition-format) can also be passed on the command line as a string using the `-c` or `--config` option:
 
 ```shell
 c2patool -c '{"assertions": [{"label": "org.contentauth.test", "data": {"name": "Jane Doe"}}]}'
 ```
  
-## Creating a new output image
-
-A file path for creating an output file with any added claim data  
-If the output file already exists, any C2PA data in that file will be replaced and the image maintained  
-If the output file doesn't exist, a parent file must be available for a source image  
-If you are not changing an image and just adding C2PA data, use an existing output file and no parent  
-If you have edited an image and want to add C2PA data to it, pass the original as the parent
-and put the edited file at the output location to have the C2PA data added.
+### Adding a manifest to a file
 
-```c2patool sample/config.json -o output.jpg```
-## Overriding the parent file
+#### Writing the manifest
 
-When using a json file, the parent file can be specified by passing -p or --parent with the path to the file
-This allows adding the same manifest data to different source images
+You can add C2PA data to a file by passing a [manifest definition](#manifest-definition-format) JSON file together with a path to a JPEG or PNG file specified by the output (`-o`) flag.
 
-## Working with .c2pa manifest files
+If the output file already exists, any C2PA data in that file will be replaced and the image maintained. If the output file doesn't exist, a parent file must be available for a source image.
 
-If the extension of the output file is '.c2pa' a standalone manifest store will be written 
+#### Overriding the parent file
 
-```c2patool claim_image.jpg -o manifest.c2pa```
+When using a JSON file, the parent file can be specified by passing `-p` or `--parent` with the path to the file. This allows adding the same manifest data to different source images.
```

Comment:
should change this to "the same manifest definition with different parent images"